build-tools/packages/build-cli/src/flags.ts

```diff
@@ -191,3 +210,72 @@ export const skipCheckFlag = Flags.boolean({
 	description: "Skip all checks.",
 	exclusive: ["install", "commit", "branchCheck", "updateCheck", "policyCheck"],
 });
+
+/**
+ * A set of flags that can be used to select packages in the repo. These flags provide a common way for commands to
+ * implement package selection and filtering.
+ */
+export const selectionFlags = {
+	all: Flags.boolean({
+		description:
+			"Run on all packages and release groups. Cannot be used with --dir, --packages, or --releaseGroup.",
+		exclusive: ["dir", "packages", "releaseGroup"],
```

Comment:
Should the exclusive list for `all`, `dir`, and `packages` include `releaseGroupRoot` too?

Reply:
fixed